cts/lab/CTStests.py

```diff
@@ -1299,9 +1303,31 @@ def choose_resource(self, node, resourcelist):
                     return rsc
         return None
 
+    def get_failcount(self, node):
+        """ Check the fail count of targeted resource on given node """
+
+        (rc, lines) = self.rsh(node,
+                               "crm_failcount --quiet --query --resource %s "
```

Comment:
I haven't tested but I'm unsure if this will work... `self.fail_resource()` calls `crm_resource -F`, which creates an asyncmon failure. Is that going to match against the operation and interval here?

Reply:
Yep, they're actually used like constants (one more thing to improve in the future)

Reply:
Okay yeah, `self.action` is always (in existing cases) set to `"asyncmon"` and `self.interval` is always set to `0`. Maybe I thought it was more general without looking closely.